shell/platform/fuchsia/flutter/pointer_delegate.cc

```diff
@@ -287,22 +287,37 @@ flutter::PointerData CreateMouseDraft(const fup_MouseEvent& event,
     ptr.buttons = flutter_buttons;
   }
 
-  // Fuchsia currently provides scroll data in "ticks", not physical pixels.
-  // However, Flutter expects scroll data in physical pixels. To compensate for
-  // lack of guidance, we make up a "reasonable amount".
-  // TODO(fxbug.dev/85388): Replace with physical pixel scroll.
+  // TODO(fxbug.dev/103443): Fuchsia previously only provides scroll data in
+  // "ticks", not physical pixels. Keep the tick bases scrolling and remove
+  // after the transistion done.
```

Comment:
```suggestion

  // Fuchsia previously only provided scroll data in "ticks", not physical pixels.
  // In these legacy situations, since Flutter expects scroll data in physical pixels,
  // to compensate for lack of guidance, we make up a "reasonable amount".
  // TODO(fxbug.dev/103443): Remove the tick based scrolling after the transition.
```